package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "mdn-browser-compat-data",
+  "version": "0.0.0",
+  "description": "Browser compatibility data provided by the Mozilla Developer Network",
+  "main": "index.js",
+  "dependencies": {
+    "extend": "3.0.1"
+  },
+  "scripts": {
+    "test": "npm test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mdn/browser-compat-data.git"
+  },
+  "keywords": [
+    "browser-compat-data",
+    "browser",
+    "compatibility",
+    "data",
+    "mdn",
+    "mozilla"
+  ],
+  "author": "mozilla",
```

Comment:
lets use `"author": "Mozilla Developer Network"` just like in https://github.com/mdn/data/blob/master/package.json